.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: E2E Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: e2e
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create workspace
+        id: workspace
+        uses:  TakeScoop/terraform-cloud-workspace-action@v2
+        with:
+          terraform_organization: takescoop-oss
+          terraform_token: ${{ secrets.TF_TOKEN_OSS }}
+          name: remote-state-action-e2e-test
+          apply: true
+          working_directory: test-e2e/
+          vcs_type: github
+
+      - name: Run action
```

Comment:
There's a race condition between this and the expected side effect of the previous step, which is a `terraform apply` and the update of the test outputs. Seems worthwhile to put this workspace in local execution mode and run `terraform apply` in the test dir.

Reply:
Which also explains why the test is already committed and not presented for review here 😉

Reply:
Still missing `terraform apply` in here, after `Create workspace`

Reply:
ah got it, so local as a setting for the workspace but handled automatically by ci 👍

Reply:
Right, this is a case where we want the pipeline and don't really care about staying "in sync" with the repo

Reply:
Speaking of, either the workspace name needs to be randomized or PR numbered and then cleaned up, or concurrency must be limited:

https://docs.github.com/en/actions/using-jobs/using-concurrency

Reply:
good idea. updated the flow to use a dynamic workspace and remove it at the end